### PR TITLE
Attn2

### PR DESCRIPTION
**Pull request by hlky**

### CODE REVIEW

**Comment by hlky** [comment]
as discussed

I can try to get it working on 24gb cards but we really need A40/A6000 or A100

**Comment by hlky** [comment]
`| 0/2000000 [00:23<?, ?it/s, loss=nan, acc=0, curr_loss=nan, curr_acc=0, ppx=nan, lr=1.2e-5, grad_norm=nan]`

Depending on what I tried 24gb OOM at either backward, optimizer.step or `sampled = decode(vqmodel, sampled)`

**Comment by hlky** [comment]
6xA5000

accelerate config using all gpu and bf16 

Also note: I was using batch size 1 🙃
